tsconfig.base.json

```diff
@@ -277,6 +277,96 @@
         "libs/gf/util-browser/src/index.ts"
       ],
       "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"],
+      "@energinet-datahub/watt/badge": [
+        "libs/ui-watt/src/lib/components/badge/index.ts"
+      ],
+      "@energinet-datahub/watt/breakpoints": [
+        "libs/ui-watt/src/lib/foundations/breakpoints/index.ts"
+      ],
+      "@energinet-datahub/watt/button": [
+        "libs/ui-watt/src/lib/components/button/index.ts"
+      ],
+      "@energinet-datahub/watt/card": [
+        "libs/ui-watt/src/lib/components/card/index.ts"
+      ],
+      "@energinet-datahub/watt/checkbox": [
+        "libs/ui-watt/src/lib/components/checkbox/index.ts"
+      ],
+      "@energinet-datahub/watt/chips": [
+        "libs/ui-watt/src/lib/components/chips/index.ts"
+      ],
+      "@energinet-datahub/watt/color": [
+        "libs/ui-watt/src/lib/foundations/color/index.ts"
```

Comment:
issue: the `index.ts` file does not exist in `color` directory.

Reply:
Yes they are, I have talked briefly with @ManBearTM that a step 2 could be converting each part of Watt being a lib on it’s own.